components/MaskedTransition/src/private/MDCMaskedTransition.m

```diff
@@ -46,83 +46,118 @@ static inline CGFloat LengthOfVector(CGVector vector) {
   return (CGFloat)hypot(vector.dx, vector.dy);
 }
 
-@interface MDCMaskedTransition () <MDMTransitionWithPresentation, MDMTransitionWithFeasibility>
-@end
+// TODO: Pull this out to MotionTransitioning.
+static NSArray <UIViewController *> *
+PrepareTransitionWithContext(id<UIViewControllerContextTransitioning> transitionContext,
+                             MDMTransitionDirection direction) {
+  UIViewController *from =
```

Comment:
Nit/naming: Maybe `from` should be `fromContext` or something similar?

Reply:
Done.